src/libexpr/eval.hh

```diff
@@ -532,12 +532,12 @@ public:
     StorePath coerceToStorePath(const PosIdx pos, Value & v, NixStringContext & context, std::string_view errorCtx);
 
     /**
-     * Part of `coerceToDerivedPath()` without any store IO which is exposed for unit testing only.
+     * Part of `coerceToSingleDerivedPath()` without any store IO which is exposed for unit testing only.
```

Comment:
What would the check be?

Reply:
It would be reading the derivation to check that it in fact has the outputs requested. But do we want to improve the error message for this in this already large PR? I think the division of labor between these functions did not change, right? So improving this message seems orthogonal?

Reply:
https://github.com/NixOS/nix/issues/8780 made and assigned to self.